src/bojack/commands/command.cr

```diff
@@ -0,0 +1,19 @@
+require "../memory"
+
+module Bojack
+  module Commands
+    # Represents a Bojack command
+    #
+    # It is a interface that each command
```

Comment:
Great! Let me change it. It is exactly what I tried to represent with this class.